daft/udf.py

```diff
@@ -146,10 +162,10 @@ def get_args_for_slice(start: int, end: int):
 
     # Post-processing of results into a Series of the appropriate dtype
     if isinstance(results[0], Series):
-        result_series = Series.concat(results)
+        result_series = Series.concat(results)  # type: ignore
         return result_series.rename(name).cast(return_dtype)._series
     elif isinstance(results[0], list):
-        result_list = [x for res in results for x in res]
+        result_list = [x for res in results for x in res]  # type: ignore
```

Comment:
MyPy isn't convinced that `results` is a list of Series or list of lists with the `isinstance` checks on its first member. This only arose because I made the `func` argument type in `run_udf` more specific. I think it's worth it though